packages/web-components/fast-foundation/src/disclosure/disclosure.template.ts

```diff
@@ -0,0 +1,13 @@
+import { html } from "@microsoft/fast-element";
+import { Disclosure } from "./disclosure";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#Disclosure} component.
+ * @public
+ */
+export const DisclosureTemplate = html<Disclosure>`
+    <template>
+        <slot name="invoker"></slot>
+        <slot name="content"></slot>
```

Comment:
Hmm, why not use the default slot so this can just nest? `<slot></slot>`

Reply:
Yes, I am going to change it https://github.com/microsoft/fast/pull/3920#discussion_r490782451

Reply:
Done :)